app/controllers/people_controller.rb

```diff
@@ -41,9 +41,11 @@ def person
   end
 
   def export
-    odt_file = Odt::Cv.new(entry).export
+    anon = params[:anon].presence || 'false'
+    odt_file = Odt::Cv.new(entry, anon).export
+    filename = anon != 'false' ? 'anonymized_cv.odt' : filename(entry.name, 'cv')
```

Comment:
That's not what i meant, just pass all params like `odt_file = Odt::Cv.new(entry, params).export`.
Also why use such a weird statement? Can't you just use `params[:anon].present?`? booleans as strings are probably always a bad idea

Reply:
but wouldn't checking if anon is present lead to unexpected behavior when calling `anon=false`?

Reply:
Are you doing that? I thought it would either send with the param being true or not present at all.
But if you do then yes it would, change it `anon == 'true'` then though to be more readable